{"levelname": "ERROR", "asctime": "2023-12-05T18:48:37.005Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 69, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "da92e18b-0324-40ee-a089-cd9a219ee90b", "collection_version_id": "f6f28606-d7a3-47c8-b4b0-3b4605789002", "errors": ["error message"]}
